--- origsrc/gnome-chemistry-utils-0.10.1/libs/canvas/Makefile.am	2008-11-15 03:12:12.000000000 -0600
+++ src/gnome-chemistry-utils-0.10.1/libs/canvas/Makefile.am	2008-11-17 13:46:57.241113100 -0600
@@ -25,5 +25,5 @@ libgcpcanvas_@GCU_API_VER@_la_SOURCES =	
 	gcp-canvas-rect-ellipse.c \
 	gcp-canvas-polygon.c
 
-libgcpcanvas_@GCU_API_VER@_la_LDFLAGS = - -version-info  @GCU_VERSION_INFO@ @GCU_LIBS@
+libgcpcanvas_@GCU_API_VER@_la_LDFLAGS = -no-undefined -version-info @GCU_VERSION_INFO@ @GCU_LIBS@
 AM_CPPFLAGS = @GCU_CFLAGS@
--- origsrc/gnome-chemistry-utils-0.10.1/libs/gcp/Makefile.am	2008-11-15 03:12:12.000000000 -0600
+++ src/gnome-chemistry-utils-0.10.1/libs/gcp/Makefile.am	2008-11-17 13:47:20.241113100 -0600
@@ -88,7 +88,7 @@ noinst_HEADERS = \
 		theme.h \
 		Hposdlg.h
 		
-libgchempaint_@GCU_API_VER@_la_LDFLAGS = -version-info  @GCU_VERSION_INFO@ \
+libgchempaint_@GCU_API_VER@_la_LDFLAGS = -no-undefined -version-info @GCU_VERSION_INFO@ \
 	$(top_builddir)/libs/gcu/libgcu-@GCU_API_VER@.la \
 	$(top_builddir)/libs/canvas/libgcpcanvas-@GCU_API_VER@.la \
 	@GCU_LIBS@
--- origsrc/gnome-chemistry-utils-0.10.1/libs/gcu/Makefile.am	2008-11-15 03:12:13.000000000 -0600
+++ src/gnome-chemistry-utils-0.10.1/libs/gcu/Makefile.am	2008-11-17 13:46:27.491113100 -0600
@@ -1,7 +1,7 @@
 MAINTAINERCLEANFILES = Makefile.in
 
 INCLUDES = -I$(top_srcdir) -I$(top_srcdir)/libs @GCU_CFLAGS@
-AM_LDFLAGS = -version-info  @GCU_VERSION_INFO@ @GCU_LIBS@
+AM_LDFLAGS = -version-info @GCU_VERSION_INFO@ -no-undefined @GCU_LIBS@
 DEFS += -DDATADIR=\"$(datadir)\" \
 	-DLOCALEDIR=\"$(localedir)\" \
 	-DPKGDATADIR=\"$(datadir)/gchemutils/@GCU_API_VER@\" \
--- origsrc/gnome-chemistry-utils-0.10.3/plugins/loaders/cdx/Makefile.am	2009-01-05 13:09:49.000000000 -0600
+++ src/gnome-chemistry-utils-0.10.3/plugins/loaders/cdx/Makefile.am	2009-01-12 00:46:51.702750000 -0600
@@ -11,6 +11,7 @@ gcu_loader_cdxdir = $(libdir)/gchemutils
 gcu_loader_cdx_LTLIBRARIES = cdx.la 
 
 cdx_la_LDFLAGS = -module -avoid-version -no-undefined
+cdx_la_LIBADD = $(top_builddir)/libs/gcu/libgcu-@GCU_API_VER@.la $(GCU_LIBS)
 
 cdx_la_SOURCES =	\
 	cdx.cc
--- origsrc/gnome-chemistry-utils-0.10.3/plugins/loaders/cdxml/Makefile.am	2009-01-05 13:09:49.000000000 -0600
+++ src/gnome-chemistry-utils-0.10.3/plugins/loaders/cdxml/Makefile.am	2009-01-12 00:48:15.515250000 -0600
@@ -11,6 +11,7 @@ gcu_loader_cdxmldir = $(libdir)/gchemuti
 gcu_loader_cdxml_LTLIBRARIES = cdxml.la 
 
 cdxml_la_LDFLAGS = -module -avoid-version -no-undefined
+cdxml_la_LIBADD = $(top_builddir)/libs/gcu/libgcu-@GCU_API_VER@.la $(GCU_LIBS)
 
 cdxml_la_SOURCES =	\
 	cdxml.cc
--- origsrc/gnome-chemistry-utils-0.10.3/plugins/paint/arrows/Makefile.am	2009-01-05 13:09:50.000000000 -0600
+++ src/gnome-chemistry-utils-0.10.3/plugins/paint/arrows/Makefile.am	2009-01-12 01:13:09.499625000 -0600
@@ -32,7 +32,11 @@ VARIABLES = \
 arrowsdir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 arrows_LTLIBRARIES = arrows.la
 
-arrows_la_LDFLAGS = @GCU_LIBS@ -module -avoid-version
+arrows_la_LDFLAGS = -module -avoid-version -no-undefined
+arrows_la_LIBADD = \
+	$(top_builddir)/libs/gcp/libgchempaint-@GCU_API_VER@.la \
+	$(top_builddir)/libs/gcu/libgcu-@GCU_API_VER@.la \
+	$(GCU_LIBS)
 
 arrows_la_SOURCES = \
 	plugin.cc \
--- origsrc/gnome-chemistry-utils-0.10.3/plugins/paint/atoms/Makefile.am	2009-01-05 13:09:49.000000000 -0600
+++ src/gnome-chemistry-utils-0.10.3/plugins/paint/atoms/Makefile.am	2009-01-12 01:13:34.046500000 -0600
@@ -27,7 +27,11 @@ VARIABLES = \
 atomsdir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 atoms_LTLIBRARIES = atoms.la
 
-atoms_la_LDFLAGS = @GCU_LIBS@ -module -avoid-version
+atoms_la_LDFLAGS = -module -avoid-version -no-undefined
+atoms_la_LIBADD = \
+	$(top_builddir)/libs/gcp/libgchempaint-@GCU_API_VER@.la \
+	$(top_builddir)/libs/gcu/libgcu-@GCU_API_VER@.la \
+	$(GCU_LIBS)
 
 atoms_la_SOURCES = \
 	plugin.cc \
--- origsrc/gnome-chemistry-utils-0.10.3/plugins/paint/bonds/Makefile.am	2009-01-05 13:09:50.000000000 -0600
+++ src/gnome-chemistry-utils-0.10.3/plugins/paint/bonds/Makefile.am	2009-01-12 01:13:46.093375000 -0600
@@ -32,7 +32,11 @@ VARIABLES = \
 bondsdir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 bonds_LTLIBRARIES = bonds.la
 
-bonds_la_LDFLAGS = @GCU_LIBS@ -module -avoid-version
+bonds_la_LDFLAGS =  -module -avoid-version -no-undefined
+bonds_la_LIBADD = \
+	$(top_builddir)/libs/gcp/libgchempaint-@GCU_API_VER@.la \
+	$(top_builddir)/libs/gcu/libgcu-@GCU_API_VER@.la \
+	$(GCU_LIBS)
 
 bonds_la_SOURCES = \
 	plugin.cc \
--- origsrc/gnome-chemistry-utils-0.10.3/plugins/paint/cycles/Makefile.am	2009-01-05 13:09:49.000000000 -0600
+++ src/gnome-chemistry-utils-0.10.3/plugins/paint/cycles/Makefile.am	2009-01-12 01:13:55.890250000 -0600
@@ -32,7 +32,11 @@ VARIABLES = \
 cyclesdir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 cycles_LTLIBRARIES = cycles.la
 
-cycles_la_LDFLAGS = @GCU_LIBS@ -module -avoid-version
+cycles_la_LDFLAGS = -module -avoid-version -no-undefined
+cycles_la_LIBADD = \
+	$(top_builddir)/libs/gcp/libgchempaint-@GCU_API_VER@.la \
+	$(top_builddir)/libs/gcu/libgcu-@GCU_API_VER@.la \
+	$(GCU_LIBS)
 
 cycles_la_SOURCES = \
 	plugin.cc \
--- origsrc/gnome-chemistry-utils-0.10.3/plugins/paint/residues/Makefile.am	2009-01-05 13:09:49.000000000 -0600
+++ src/gnome-chemistry-utils-0.10.3/plugins/paint/residues/Makefile.am	2009-01-12 01:14:08.937125000 -0600
@@ -9,7 +9,11 @@ DEFS += -DDATADIR=\"$(datadir)\" \
 residuesdir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 residues_LTLIBRARIES = residues.la
 
-residues_la_LDFLAGS = @GCU_LIBS@ -module -avoid-version
+residues_la_LDFLAGS = -module -avoid-version -no-undefined
+residues_la_LIBADD = \
+	$(top_builddir)/libs/gcp/libgchempaint-@GCU_API_VER@.la \
+	$(top_builddir)/libs/gcu/libgcu-@GCU_API_VER@.la \
+	$(GCU_LIBS)
 
 residues_la_SOURCES = \
 	plugin.cc \
--- origsrc/gnome-chemistry-utils-0.10.3/plugins/paint/selection/Makefile.am	2009-01-05 13:09:50.000000000 -0600
+++ src/gnome-chemistry-utils-0.10.3/plugins/paint/selection/Makefile.am	2009-01-12 01:14:19.109000000 -0600
@@ -32,7 +32,11 @@ VARIABLES = \
 selectiondir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 selection_LTLIBRARIES = selection.la
 
-selection_la_LDFLAGS = @GCU_LIBS@ -module -avoid-version
+selection_la_LDFLAGS = -module -avoid-version -no-undefined
+selection_la_LIBADD = \
+	$(top_builddir)/libs/gcp/libgchempaint-@GCU_API_VER@.la \
+	$(top_builddir)/libs/gcu/libgcu-@GCU_API_VER@.la \
+	$(GCU_LIBS)
 
 selection_la_SOURCES = \
 	plugin.cc \
--- origsrc/gnome-chemistry-utils-0.10.3/plugins/paint/templates/Makefile.am	2009-01-05 13:09:50.000000000 -0600
+++ src/gnome-chemistry-utils-0.10.3/plugins/paint/templates/Makefile.am	2009-01-12 01:14:30.030875000 -0600
@@ -9,7 +9,11 @@ DEFS += -DDATADIR=\"$(datadir)\" \
 templatesdir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 templates_LTLIBRARIES = templates.la
 
-templates_la_LDFLAGS = @GCU_LIBS@ -module -avoid-version
+templates_la_LDFLAGS = -module -avoid-version -no-undefined
+templates_la_LIBADD = \
+	$(top_builddir)/libs/gcp/libgchempaint-@GCU_API_VER@.la \
+	$(top_builddir)/libs/gcu/libgcu-@GCU_API_VER@.la \
+	$(GCU_LIBS)
 
 templates_la_SOURCES = \
 	plugin.cc \
--- origsrc/gnome-chemistry-utils-0.10.3/plugins/paint/text/Makefile.am	2009-01-05 13:09:49.000000000 -0600
+++ src/gnome-chemistry-utils-0.10.3/plugins/paint/text/Makefile.am	2009-01-12 01:14:40.109000000 -0600
@@ -22,7 +22,11 @@ VARIABLES = \
 textdir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 text_LTLIBRARIES = text.la
 
-text_la_LDFLAGS = @GCU_LIBS@ -module -avoid-version
+text_la_LDFLAGS = -module -avoid-version -no-undefined
+text_la_LIBADD = \
+	$(top_builddir)/libs/gcp/libgchempaint-@GCU_API_VER@.la \
+	$(top_builddir)/libs/gcu/libgcu-@GCU_API_VER@.la \
+	$(GCU_LIBS)
 
 text_la_SOURCES = \
 	plugin.cc \
--- origsrc/gnome-chemistry-utils-0.10.3/plugins/paint/wikipedia/Makefile.am	2009-01-05 13:09:50.000000000 -0600
+++ src/gnome-chemistry-utils-0.10.3/plugins/paint/wikipedia/Makefile.am	2009-01-12 01:14:50.171500000 -0600
@@ -19,7 +19,11 @@ VARIABLES = \
 wikipediadir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 wikipedia_LTLIBRARIES = wikipedia.la
 
-wikipedia_la_LDFLAGS = @GCU_LIBS@ -module -avoid-version
+wikipedia_la_LDFLAGS = -module -avoid-version -no-undefined
+wikipedia_la_LIBADD = \
+	$(top_builddir)/libs/gcp/libgchempaint-@GCU_API_VER@.la \
+	$(top_builddir)/libs/gcu/libgcu-@GCU_API_VER@.la \
+	$(GCU_LIBS)
 
 wikipedia_la_SOURCES = \
 	plugin.cc \
