--- origsrc/gnome-chemistry-utils-0.12.0/gnumeric/Makefile.am	2010-05-12 06:04:18.000000000 -0500
+++ src/gnome-chemistry-utils-0.12.0/gnumeric/Makefile.am	2010-05-17 02:04:07.167001400 -0500
@@ -14,7 +14,7 @@ DEFS += -DDATADIR=\"$(datadir)\" -DGNM_V
 
 gchemutils_plugindir = $(GNUMERIC_PLUGINS_DIR)/gchemutils
 gchemutils_plugin_LTLIBRARIES = plugin.la 
-plugin_la_LDFLAGS = -module -avoid-version
+plugin_la_LDFLAGS = -module -avoid-version -no-undefined
 plugin_la_SOURCES =	\
 	functions.cc
 plugin_la_LIBADD = \
--- origsrc/gnome-chemistry-utils-0.12.0/libs/gccv/Makefile.am	2010-05-12 06:04:15.000000000 -0500
+++ src/gnome-chemistry-utils-0.12.0/libs/gccv/Makefile.am	2010-05-17 00:36:44.410132800 -0500
@@ -48,7 +48,7 @@ libgccv_@GCU_API_VER@_la_SOURCES =	\
 	wedge.cc
 
 libgccv_@GCU_API_VER@_la_LDFLAGS = \
-		-version-info  @GCU_VERSION_INFO@
+		-version-info  @GCU_VERSION_INFO@ -no-undefined
 libgccv_@GCU_API_VER@_la_LIBADD = \
 		$(cairo_LIBS) \
 		$(gtk_LIBS) \
--- origsrc/gnome-chemistry-utils-0.12.0/libs/gcp/Makefile.am	2010-05-12 06:04:17.000000000 -0500
+++ src/gnome-chemistry-utils-0.12.0/libs/gcp/Makefile.am	2010-05-17 00:36:51.618545100 -0500
@@ -110,7 +110,7 @@ noinst_HEADERS = \
 		Hposdlg.h
 		
 libgcp_@GCU_API_VER@_la_LDFLAGS = \
-		-version-info  @GCU_VERSION_INFO@
+		-version-info  @GCU_VERSION_INFO@ -no-undefined
 libgcp_@GCU_API_VER@_la_LIBADD = \
 		$(top_builddir)/libs/gcu/libgcu-@GCU_API_VER@.la \
 		$(top_builddir)/libs/gccv/libgccv-@GCU_API_VER@.la
--- origsrc/gnome-chemistry-utils-0.12.0/libs/gcu/Makefile.am	2010-05-12 06:04:17.000000000 -0500
+++ src/gnome-chemistry-utils-0.12.0/libs/gcu/Makefile.am	2010-05-17 00:36:32.583456400 -0500
@@ -7,7 +7,7 @@ INCLUDES = \
 		$(openbabel_CFLAGS) \
 		$(GCU_CFLAGS)
 libgcu_@GCU_API_VER@_la_LDFLAGS = \
-		-version-info  @GCU_VERSION_INFO@
+		-version-info  @GCU_VERSION_INFO@ -no-undefined
 libgcu_@GCU_API_VER@_la_LIBADD = \
 		$(goffice_LIBS) \
 		$(openbabel_LIBS) \
--- origsrc/gnome-chemistry-utils-0.12.8/mozilla-plugin/Makefile.am	2011-05-08 02:27:39.000000000 -0500
+++ src/gnome-chemistry-utils-0.12.8/mozilla-plugin/Makefile.am	2011-06-29 17:18:30.160131800 -0500
@@ -14,7 +14,7 @@ DEFS += -DLIBEXECDIR=\"$(libexecdir)\" -
 
 libmozgcu_la_SOURCES = moz-plugin.c
 libmozgcu_la_CPPFLAGS = $(MOZILLA_CFLAGS)
-libmozgcu_la_LDFLAGS = -module -avoid-version 
+libmozgcu_la_LDFLAGS = -module -avoid-version -no-undefined
 libmozgcu_la_LIBADD = $(MOZILLA_LIBS)
 
 chem_viewer_SOURCES = chem-viewer.cc
--- origsrc/gnome-chemistry-utils-0.12.0/plugins/paint/arrows/Makefile.am	2010-05-12 06:04:11.000000000 -0500
+++ src/gnome-chemistry-utils-0.12.0/plugins/paint/arrows/Makefile.am	2010-05-17 00:36:32.592456900 -0500
@@ -37,7 +37,7 @@ VARIABLES = \
 arrowsdir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 arrows_LTLIBRARIES = arrows.la
 
-arrows_la_LDFLAGS = -module -avoid-version
+arrows_la_LDFLAGS = -module -avoid-version -no-undefined
 arrows_la_LIBADD = \
 		$(top_builddir)/libs/gcp/libgcp-@GCU_API_VER@.la
 
--- origsrc/gnome-chemistry-utils-0.12.0/plugins/paint/atoms/Makefile.am	2010-05-12 06:04:11.000000000 -0500
+++ src/gnome-chemistry-utils-0.12.0/plugins/paint/atoms/Makefile.am	2010-05-17 00:36:32.601457400 -0500
@@ -35,7 +35,7 @@ VARIABLES = \
 atomsdir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 atoms_LTLIBRARIES = atoms.la
 
-atoms_la_LDFLAGS = -module -avoid-version
+atoms_la_LDFLAGS = -module -avoid-version -no-undefined
 atoms_la_LIBADD = \
 		$(top_builddir)/libs/gcp/libgcp-@GCU_API_VER@.la
 
--- origsrc/gnome-chemistry-utils-0.12.0/plugins/paint/bonds/Makefile.am	2010-05-12 06:04:11.000000000 -0500
+++ src/gnome-chemistry-utils-0.12.0/plugins/paint/bonds/Makefile.am	2010-05-17 00:36:32.610457900 -0500
@@ -39,7 +39,7 @@ VARIABLES = \
 bondsdir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 bonds_LTLIBRARIES = bonds.la
 
-bonds_la_LDFLAGS = -module -avoid-version
+bonds_la_LDFLAGS = -module -avoid-version -no-undefined
 bonds_la_LIBADD = \
 		$(top_builddir)/libs/gcp/libgcp-@GCU_API_VER@.la
 
--- origsrc/gnome-chemistry-utils-0.12.0/plugins/paint/cycles/Makefile.am	2010-05-12 06:04:11.000000000 -0500
+++ src/gnome-chemistry-utils-0.12.0/plugins/paint/cycles/Makefile.am	2010-05-17 00:36:32.621458600 -0500
@@ -37,7 +37,7 @@ VARIABLES = \
 cyclesdir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 cycles_LTLIBRARIES = cycles.la
 
-cycles_la_LDFLAGS = -module -avoid-version
+cycles_la_LDFLAGS = -module -avoid-version -no-undefined
 cycles_la_LIBADD = \
 		$(top_builddir)/libs/gcp/libgcp-@GCU_API_VER@.la
 
--- origsrc/gnome-chemistry-utils-0.12.0/plugins/paint/residues/Makefile.am	2010-05-12 06:04:11.000000000 -0500
+++ src/gnome-chemistry-utils-0.12.0/plugins/paint/residues/Makefile.am	2010-05-17 00:36:32.631459100 -0500
@@ -14,7 +14,7 @@ DEFS += -DDATADIR=\"$(datadir)\" \
 residuesdir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 residues_LTLIBRARIES = residues.la
 
-residues_la_LDFLAGS = -module -avoid-version
+residues_la_LDFLAGS = -module -avoid-version -no-undefined
 residues_la_LIBADD = \
 		$(top_builddir)/libs/gcp/libgcp-@GCU_API_VER@.la
 
--- origsrc/gnome-chemistry-utils-0.12.0/plugins/paint/selection/Makefile.am	2010-05-12 06:04:11.000000000 -0500
+++ src/gnome-chemistry-utils-0.12.0/plugins/paint/selection/Makefile.am	2010-05-17 00:36:32.640459700 -0500
@@ -37,7 +37,7 @@ VARIABLES = \
 selectiondir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 selection_LTLIBRARIES = selection.la
 
-selection_la_LDFLAGS = -module -avoid-version
+selection_la_LDFLAGS = -module -avoid-version -no-undefined
 selection_la_LIBADD = \
 		$(top_builddir)/libs/gcp/libgcp-@GCU_API_VER@.la
 
--- origsrc/gnome-chemistry-utils-0.12.0/plugins/paint/templates/Makefile.am	2010-05-12 06:04:12.000000000 -0500
+++ src/gnome-chemistry-utils-0.12.0/plugins/paint/templates/Makefile.am	2010-05-17 00:36:32.649460200 -0500
@@ -14,7 +14,7 @@ DEFS += -DDATADIR=\"$(datadir)\" \
 templatesdir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 templates_LTLIBRARIES = templates.la
 
-templates_la_LDFLAGS = -module -avoid-version
+templates_la_LDFLAGS = -module -avoid-version -no-undefined
 templates_la_LIBADD = \
 		$(top_builddir)/libs/gcp/libgcp-@GCU_API_VER@.la
 
--- origsrc/gnome-chemistry-utils-0.12.0/plugins/paint/text/Makefile.am	2010-05-12 06:04:11.000000000 -0500
+++ src/gnome-chemistry-utils-0.12.0/plugins/paint/text/Makefile.am	2010-05-17 00:36:32.659460700 -0500
@@ -27,7 +27,7 @@ VARIABLES = \
 textdir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 text_LTLIBRARIES = text.la
 
-text_la_LDFLAGS = -module -avoid-version
+text_la_LDFLAGS = -module -avoid-version -no-undefined
 text_la_LIBADD = \
 		$(top_builddir)/libs/gcp/libgcp-@GCU_API_VER@.la
 
--- origsrc/gnome-chemistry-utils-0.12.0/plugins/paint/wikipedia/Makefile.am	2010-05-12 06:04:11.000000000 -0500
+++ src/gnome-chemistry-utils-0.12.0/plugins/paint/wikipedia/Makefile.am	2010-05-17 00:36:32.667461200 -0500
@@ -24,7 +24,7 @@ VARIABLES = \
 wikipediadir = $(libdir)/gchemutils/@GCU_API_VER@/plugins/paint
 wikipedia_LTLIBRARIES = wikipedia.la
 
-wikipedia_la_LDFLAGS = -module -avoid-version
+wikipedia_la_LDFLAGS = -module -avoid-version -no-undefined
 wikipedia_la_LIBADD = \
 		$(top_builddir)/libs/gcp/libgcp-@GCU_API_VER@.la
 
